doc/02-Installation.md

```diff
@@ -0,0 +1,125 @@
+host  all icinga-notifications      ::/0 md5
+```
+
+To apply these changes, run `systemctl reload postgresql`.
+
+After creating the database, import the Icinga Notifications schema using the following command:
+
+```
+psql -U icinga-notifications icinga-notifications < /usr/share/icinga-notifications/schema/pgsql/schema.sql
+```
+
+## Configuring Icinga Notifications
+
+Icinga Notifications installs its configuration file to `/etc/icinga-notifications/config.yml`,
+pre-populating most of the settings for a local setup. Before running Icinga Notifications,
+adjust the database credentials and, if necessary, the Icinga Web 2 URL.
+The configuration file explains general settings.
+All available settings can be found under [Configuration](03-Configuration.md).
+
+## Running Icinga Notifications
+
+The `icinga-notifications` package automatically installs the necessary systemd unit files to run Icinga Notifications.
+Please run the following command to enable and start its service:
+
+```bash
+systemctl enable --now icinga-notifications
+```
+
+## Installing Icinga Notifications Web
+
+With Icinga 2, Icinga Notifications and the database fully set up, it is now time to install Icinga Notifications Web,
+which connects to the database and allows configuring Icinga Notifications.
+
+The Icinga Notifications Web package is also included in the Icinga repository, and since it is already set up,
+you have completed the instructions here and can proceed to
+<!-- {% if amazon_linux %} -->
+[installing Icinga Notifications Web on Amazon Linux](https://icinga.com/docs/icinga-notifications-web/latest/doc/02-Installation/01-Amazon-Linux/#installing-icinga-notifications-web-package),
+<!-- {% endif %} -->
+<!-- {% if centos %} -->
+[installing Icinga Notifications Web on CentOS](https://icinga.com/docs/icinga-notifications-web/latest/doc/02-Installation/02-CentOS/#installing-icinga-notifications-web-package),
+<!-- {% endif %} -->
+<!-- {% if debian %} -->
+[installing Icinga Notifications Web on Debian](https://icinga.com/docs/icinga-notifications-web/latest/doc/02-Installation/03-Debian/#installing-icinga-notifications-web-package),
+<!-- {% endif %} -->
+<!-- {% if rhel %} -->
+[installing Icinga Notifications Web on RHEL](https://icinga.com/docs/icinga-notifications-web/latest/doc/02-Installation/04-RHEL/#installing-icinga-notifications-web-package),
+<!-- {% endif %} -->
+<!-- {% if sles %} -->
+[installing Icinga Notifications Web on SLES](https://icinga.com/docs/icinga-notifications-web/latest/doc/02-Installation/05-SLES/#installing-icinga-notifications-web-package),
+<!-- {% endif %} -->
+<!-- {% if ubuntu %} -->
+[installing Icinga Notifications Web on Ubuntu](https://icinga.com/docs/icinga-notifications-web/latest/doc/02-Installation/06-Ubuntu/#installing-icinga-notifications-web-package),
+<!-- {% endif %} -->
+which will also guide you through the setup of the Icinga Web PHP framework,
+which is required to run the Icinga Notifications web module.
```

Comment:
> Icinga Web PHP framework

I haven't heard that name before. Does that mean IPL? In anyway, sounds like a level of detail that wouldn't be necessary here anyway and something like "its dependencies" should suffice.